### PR TITLE
Add in new content to the how-to page

### PR DESCRIPTION
**Pull request by lamtu1**
`In Progress:` Add in content on the `how-to` page with new content and layout.

### CODE REVIEW

**Comment by lamtu1** [comment]
`Fixed for Issue:`   #2101

**Comment by lamtu1** [comment]
`Quick Video:` Demoing the How-To Use page
https://github.com/OregonDigital/OD2/assets/111306182/1cbe603b-85ce-4812-8966-9b64433f4387

**Comment by straleyb** [comment]
@lamtu1 Is this ready to go or still WIP?

**Comment by lamtu1** [comment]
@straleyb I think it is ready to go. We can have this up for now, and if anyone have anymore suggestion or edit, I can alter it later on.

File: app/views/oregon_digital/about/use.html.erb [hunk 1]
```diff
@@ -1,13 +1,76 @@
+<html id="use_page">
+<body>
 <h2>How to Use Oregon Digital</h2>
 
-<h3>Searching and Faceting</h3>
+<br />
+
+<%# LIST: Showing Table of Content for navigation %>
+<h3>Table of Contents</h3>
+<ol class="use_toc">
```
Comment (CGillen) [inline app/views/oregon_digital/about/use.html.erb:9]: Good use of `ol`

**Review by CGillen — CHANGES_REQUESTED** [review]
What was your reasoning for styling this page different than the rest of the about pages?

**Comment by lamtu1** [comment]
> What was your reasoning for styling this page different than the rest of the about pages?

I did this styling different as I was following along the google docs requirement on the ticket. Here is the link to the [Google Doc](https://docs.google.com/document/d/17xaOFMCH-_5s-txlTIzroFF3369Q-chAm-EGau5Oay8/edit#heading=h.byqkkjiuntp6).

I was not sure what the stylization is for the about page, so I just layout based on this requirement as follow in the link above.

**Comment by CGillen** [comment]
The layout looks great, but I don't see where they mentioned they needed different CSS-type styling.
You should also almost never use `display: table` It usually means you're either working with a hack, or have messed up your `<table>` somehow. Either way, it's really bad for a11y

**Comment by lamtu1** [comment]
@CGillen Ah I see, I was following a tutorial on how to use CSS to display the number like in the demo video on the table of content. And I created the CSS page to isolate the content only on that page out like some spacing issue between `<p>` and `<h3> or <h2>` tag and make it only affecting on that `how-to` page. But I can rework it on the `display: table`and if we are sticking with no CSS page, I can rework it as well to use our CSS only.

**Comment by CGillen** [comment]
If you need a separate css file just for the how-to page, that's fine. Just try to be aware of the font/text styling, to keep it consistent with the rest of the site. There should be some helper classes for some common resizing/styling fonts here
https://github.com/OregonDigital/OD2/blob/master/app/assets/stylesheets/oregon_digital/fonts.scss

stuff like `.lato-p`, or `.mw-lg`. Those aren't ones I specifically want you to use somewhere, they're just examples of the kinds of classes that we commonly use to keep fonts consistent

**Comment by lamtu1** [comment]
I will do that. I will fix that `display: table` in the css and beware of the other pages, and look how to apple consistency based on the `fonts.scss` file. Thank you for the feedback @CGillen

**Comment by CGillen** [comment]
Can you post another screenshot? I'm worried some text is still serif and that just doesn't feel right. All font should be `lato`

**Comment by lamtu1** [comment]
Here some screenshot

Screenshot 2023-08-02 at 2 39 30 PM

Screenshot 2023-08-02 at 2 39 45 PM

**Review by CGillen — APPROVED** [review]

**Pull request merged**